src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1067,20 +1067,17 @@
      *
      * <p>For example:</p>
      * <pre>
-     * this(1);
+     * |--this -&gt; 1
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * +--CTOR_CALL (this)
-     *     |
-     *     +--LPAREN (()
-     *     +--ELIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (1)
-     *     +--RPAREN ())
-     *     +--SEMI (;)
+     * |--CTOR_CALL -&gt; CTOR_CALL
```

Comment:
Please remove all leftmost `|`.  The fragment should start from `CTOR_CALL`.
Since the `SEMI` token is a child of `CTOR_CALL`, no siblings of `CTOR_CALL` are expected here.

For cases like [this](https://github.com/checkstyle/checkstyle/pull/9431/files#diff-ba93830ba2cf23d542b30560fb43bdc8d8d1be05025e84ee6a96cca6ef8e3342) where `SEMI` is a sibling, the leftmost column with `|` is needed.